### DISCUSSION TITLE
One more time :( low light and denoise video

### DISCUSSION THREAD

**Discussion by And1re** [gopro/labs#782] [Q&A] [answered]
Hello, I have already asked this question, but I will ask again :) I'm doing CQB airsoft, in a dark room. But no matter how hard I try and use video editors, I still get a dark Video without sharpness. Please help me with the QR code for this type of video shooting![image](https://github.com/gopro/labs/assets/16433915/bc8a6e1e-5996-4c39-9841-f60d3a8d2c8e)![image](https://github.com/gopro/labs/assets/16433915/999d78b6-16fb-44c2-a98b-ed809286f8c5)

**Comment by dnewman-gpsw** [comment]
If you "have already asked this question", what answers did try?

**Reply by And1re** [reply]
> If you "have already asked this question", what answers did try? 

I want to ask QR code for this type of video shooting

**Reply by And1re** [reply]
> If you "have already asked this question", what answers did try? 

Because the previous answer to my question did not give results

**Reply by dnewman-gpsw** [reply]
What was that answer?  As you didn't follow-up on the original thread, I can't determine how to help.  I can only guess you would have been advised to turn down the denoise and do that in post.

**Reply by dnewman-gpsw** [reply]
I found you original post, which was also a little vague in the question, so no clear answer was given, other than general info on exposure. You have not explained, your frame rate needs, you stabilization needs, your post production experience or really anything much. Even what camera you are using. These images aren't much do go on.

But it you want to be dropped into the deep-end:

oMNR01=1  (turn off in camera noise reduction - H11/12 only )
oMGCMP=1  (turn off gyro compensated exposure, equal more motion blur, but less noise - H11/12 only.)  
cF - Color Flat (all models)

If you are only in low-light, you use a locked shutter to about (1/192 to 1/240 for okay stabilization  -- r5ti8M1S45) or (1/48 to 1/60, if you don't need good stabilization, but want lower noise -- r5ti8M1S180.) These commands also set 5K with 800 ISO MAX.  Shoot 5k 16x9, and down-res in post (down res helps denoise.)  5k -> 1920x1080 then use NeatVideo or similar to control the noise. 

As a single QR (option 1)
![image](https://github.com/gopro/labs/assets/10856084/e4d1562b-d167-4bd0-a1e7-5674bfb2de3f)

Good stab (option 2) :
![image](https://github.com/gopro/labs/assets/10856084/b38ed83b-b45b-4257-a3be-57e5fdc990a3)

Good noise (option 3):
![image](https://github.com/gopro/labs/assets/10856084/ae7bb983-6a2c-4417-a36e-f120634a53e8)

**Reply by dnewman-gpsw** [reply]
p.s. wN - White Balance natural also help reduce noise in low light.

**Reply by And1re** [reply]
> I found you original post, which was also a little vague in the question, so no clear answer was given, other than general info on exposure. You have not explained, your frame rate needs, you stabilization needs, your post production experience or really anything much. Even what camera you are using. These images aren't much do go on.
> 
> 
> 
> But it you want to be dropped into the deep-end:
> 
> 
> 
> oMNR01=1  (turn off in camera noise reduction - H11/12 only )
> 
> oMGCMP=1  (turn off gyro compensated exposure, equal more motion blur, but less noise - H11/12 only.)  
> 
> cF - Color Flat (all models)
> 
> 
> 
> If you are only in low-light, you use a locked shutter to about (1/192 to 1/240 for okay stabilization  -- r5ti8M1S45) or (1/48 to 1/60, if you don't need good stabilization, but want lower noise -- r5ti8M1S180.) These commands also set 5K with 800 ISO MAX.  Shoot 5k 16x9, and down-res in post (down res helps denoise.)  5k -> 1920x1080 then use NeatVideo or similar to control the noise. 
> 
> 
> 
> As a single QR (option 1)
> 
> ![image](https://github.com/gopro/labs/assets/10856084/e4d1562b-d167-4bd0-a1e7-5674bfb2de3f)
> 
> 
> 
> Good stab (option 2) :
> 
> ![image](https://github.com/gopro/labs/assets/10856084/b38ed83b-b45b-4257-a3be-57e5fdc990a3)
> 
> 
> 
> Good noise (option 3):
> 
> ![image](https://github.com/gopro/labs/assets/10856084/ae7bb983-6a2c-4417-a36e-f120634a53e8)
> 
> 
> 
> 
> 
> 

Thank you for your reply! I answer you
, the frame rate is 60, 
the GoPro Hero 12 camera, stabilization is needed, since I run to the video, 
edit everything in davinci resolved, as you advised, removing the noise on the video there

**Reply by dnewman-gpsw** [reply, marked as answer]
As you are on HERO12 you could use GPLog, rather than color Flat. As you using 60fps, you could use a 90 degree shutter (if indoor only, and always low light.)

```oMNR01=1oMGCMP=0r5p60ti8M1S90cfwNsM```

This temporary command will set Noise Reduction off, gyro comp on, 5.3Kp60, ISO range 100-800, Shutter 1/240, color flat, white balance Native, sharpeness Medium.

Or 

```oMNR01=1oMGCMP=1r5p60ti8M1S0cfwNsM```

This temporary command will set Noise Reduction off, gyro comp off, 5.3Kp60, ISO range 100-800, Shutter Auto, color flat, white balance Native, sharpeness Medium.  This will be less stable, but better for really low light conditions.

In all examples you could increase ISO MAX as needed.

**Reply by And1re** [reply]
Thank you I will try
